core/src/main/java/com/hcl/ecomm/core/services/impl/CartServiceImpl.java

```diff
@@ -175,4 +194,50 @@ public String applyCoupon(String couponApplied) {
 
         return couponDiscount;
     }
-}
\ No newline at end of file
+
+    @Override
+    public JsonArray getCustomerCart(String customerToken) {
+        String token = "";
+        String url = "";
+        JSONObject getCartResponse = new JSONObject();
+        String domainName = loginService.getDomainName();
+
+            token = customerToken;
+            url = schema + "://" + domainName + config.customer_getCart_string() ;
+
+        JsonArray cartItems = null;
+        LOG.info("url : " + url);
```

Comment:
@DeepaliAgrwl : change this to log.debug

Reply:
log level has been updated. Please check